### DISCUSSION TITLE
"media_player" row: do not use circular images

### DISCUSSION THREAD

**Discussion by ildar170975** [home-assistant/frontend#16871] [Dashboard] [answered]
Here:
![изображение](https://github.com/home-assistant/frontend/assets/71872483/db480461-3946-4aec-812c-a2e31f091946)

Here:
![изображение](https://github.com/home-assistant/frontend/assets/71872483/1cf25895-484c-4762-9af1-45849944240d)

It is weird to have a circular images for covers.
Suggest to use something like `border_radius: 10%`:

![изображение](https://github.com/home-assistant/frontend/assets/71872483/a99d209b-6fa3-40e9-acaa-8f21a3f27387)

**Comment by ildar170975** [comment]
Imho at least this file should be edited:
https://github.com/home-assistant/frontend/blob/dev/src/components/entity/state-badge.ts

If the entity has an image or an image is defined for the badge, then do not use `border-radius: 50%`

**Comment by ildar170975** [comment]
Prepared a commit to add a `border-radius: 10%` style for "hui-media-player-entity-row" element.
https://github.com/ildar170975/frontend/commit/830bcc14af9612075b7055e0863735322de653f4
Cannot test it properly myself right now; would be happy if someone test it.

**Reply by Quentame** [reply]
Better create a PR, with this discussion linked

**Reply by ildar170975** [reply, marked as answer]
Created a PR: https://github.com/home-assistant/frontend/pull/18066
